charts/castai-cluster-controller/templates/deployment.yaml

```
@@ -42,6 +42,15 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       affinity:
+        podAntiAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              - labelSelector:
+                  matchExpressions:
+                    - key: app.kubernetes.io/name
+                      operator: In
+                      values:
+                      - castai-cluster-controller
```

Can you try something like this?
```suggestion
                      - {{ include "castai-agent.name" . }}
```

Source https://github.com/castai/helm-charts/blob/38c5ea89568d6789b9957865a8100516a879e880/charts/castai-cluster-controller/templates/_helpers.tpl#L47

We have some copy-paste bug in template name :D

---

Updated, that copy paste makes me cringe :-)